tests/wait_mysql_up.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+TIMEOUT=${1:-60}
```

Comment:
timeout is usually represent time, while it is used here for counting while loop cycles.

Reply:
I agree it's nicer, just not sure if it's critical since the sleep interval is hardcoded to 1 sec, the timeout is in fact number of seconds (if you disregard the time it takes for the other commands)
doing based on time would be something like:
```
startTime=$(date +%s)
while true; do
   <code here>
   timeNow=$(date +%s)
   (( secondsDiff = $timeNow - $startTime ))
   if [[ $secondsDiff -ge $TIMEOUT ]]; then
    exit 1
  fi
done
```

Reply:
I could write this code in various ways.
I also could use the `timeout` command outside the script (in .travis.yml) and use less code here.
I can't see any difference in behavior in the bottom line.

Reply:
True, but `timeout` in the travis is less DRY since it might be also required as part of compse and would be best to encapsulate it all in a single script

Reply:
without `sleep` we are running too much unnecessary loops.